packages/contracts/contracts/deploy/PayableDAppSystem.sol

```diff
@@ -16,9 +17,25 @@ contract PayableDAppSystem is IPayableDAppSystem {
     ENS public immutable ens;
     ICartesiDAppFactory public immutable factory;
 
-    constructor(ENS _ens, ICartesiDAppFactory _factory) {
+    uint256 public constant TOTAL_SHARES = 10000;
+    uint256 public immutable systemShares;
+    address public immutable payee;
+
+    constructor(
+        ENS _ens,
+        ICartesiDAppFactory _factory,
+        uint256 _systemShares,
+        address _payee
+    ) {
+        require(
+            _systemShares < TOTAL_SHARES,
```

Comment:
Why not `<=`? The system wouldn't break if the factory payee had no shares, would it?